pkg/tsdb/cloudwatch/routes/namespaces.go

```diff
@@ -0,0 +1,34 @@
+package routes
+
+import (
+	"encoding/json"
+	"net/http"
+	"net/url"
+	"sort"
+	"strings"
+
+	"github.com/grafana/grafana-plugin-sdk-go/backend"
+	"github.com/grafana/grafana/pkg/tsdb/cloudwatch/models"
+	"github.com/grafana/grafana/pkg/tsdb/cloudwatch/services"
+)
+
+func NamespacesHandler(pluginCtx backend.PluginContext, reqCtxFactory models.RequestContextFactoryFunc, _ url.Values) ([]byte, *models.HttpError) {
+	reqCtx, err := reqCtxFactory(pluginCtx, "default")
+	if err != nil {
+		return nil, models.NewHttpError("error in NamespacesHandler", http.StatusInternalServerError, err)
+	}
+
+	hardcodedNamespaces := services.GetHardCodedNamespaces()
```

Comment:
just a thought: wondering if this should be called `namespaces` or `result` or something because it's no longer just the hardcoded after custom namespaces are appended

Reply:
Yeah makes sense, will fix.

Reply:
Sorry that's not what I meant, I made rough suggestions but feel free to disregard or revert back to what you had.